images/bridge2/assistant/chat.go

```diff
@@ -0,0 +1,93 @@
+package assistant
```

Comment:
we should include a note about where this was copied from (and the associated license)